RoaringBitmap/src/main/java/org/roaringbitmap/longlong/ImmutableLongBitmapDataProvider.java

```diff
@@ -142,6 +143,22 @@ public default LongStream reverseStream() {
    */
   public long select(long j);
 
+  /**
+   * Get the first (smallest) integer in this RoaringBitmap,
+   * that is, returns the minimum of the set.
```

Comment:
```suggestion
   * that is, return the minimum of the set.
```

Reply:
Just reproduced as in RoaringBitmap. Fixing both.